templates/zerver/api/incoming-webhooks-walkthrough.md

```diff
@@ -36,6 +36,38 @@ thing, it requires only one fixture,
 }
 ```
 
+If your integration depends on HTTP headers sent by the service you are integrating with,
+then it would be a good idea to store these too. An example of when you would need to do
+this would be if the service sends the event type as a custom http header, like with
+GitHub's webhooks.
```

Comment:
The first sentence is irrelevant and doesn't add much substance. I think you can remove the first sentence and simply add a sub-heading **Event type HTTP headers**.

Reply:
@eeshangarg did you accidentally resolve this comment?